Makefile

```diff
@@ -18,7 +18,7 @@ setup: ## Configure huge-pages (requires sudo/root password)
 	@echo Actual Hugepages allocation: $$(cat $(HUGEPAGES_DIR))
 	@[ $$(cat $(HUGEPAGES_DIR)) -eq $(HUGEPAGES) ]
 
-build push pull logs: SVC ?= spdk nvmeof nvmeof-cli ceph
+build push pull logs: SVC ?= spdk nvmeof nvmeof-devel nvmeof-cli ceph
```

Comment:
We shouldn't be pushing `nvmeof-devel`... Is this just for `logs`?

Reply:
@epuertat for 'build' as well . We could separate "push" definition and remove `nvmeof-devel` and maybe `spdk` (should it be pushed?) or alternatively push also the devel image, since it appears to be an artifact of the build process and potentially useful for developers . WDYT?

Reply:
Ok. `spdk` should only be built and pushed when there are changes to the `spdk` fork... I just mentioned that to @barakda in another PR, that it'd save 10 mins of CI not to build SPDK, but just pull it.

Reply:
🖖 defined `push` default SVC ([here](https://github.com/baum/ceph-nvmeof/blob/b270634f985364b9322faeeab189ecdc03fe3cc1/mk/containerized.mk#L23)) as:
```Makefile
push: ## Push SVC container images to a registry. Requires previous "docker login"
push: SVC ?= nvmeof nvmeof-cli ceph
```